src/porepy/grids/mortar_grid.py

```diff
@@ -19,10 +18,15 @@
 class MortarGrid:
     """
     Parent class for mortar grids it contains two grids representing the left
-    and right part of the mortar grid and the weighted mapping from the higher
-    dimensional grid (as set of faces) to the mortar grids and from the lower
-    dimensional grid (as set of cells) to the mortar grids. The two mortar grids
-    can be different.
+    and right part of the mortar grid and the weighted mapping from the master
+    grid to the mortar grids and from the slave grid to the mortar grids.
+    The two mortar grids can be different. The master grid is assumed to be one
+    dimension higher than the mortar grids, while the slave grid can either one
+    dimension higher or the same dimension as the mortar grids.
+
+    NOTE: The mortar class is mostly tested for the case when the slave grid has
+    the same dimension as the mortar grid. Especially, the updating of any grid
```

Comment:
There is a sentence missing here, I think. For the errors, you are refering to the case where the two domains have the same dimension?